awx/main/models/ha.py

```diff
@@ -187,7 +187,7 @@ def get_cleanup_task_kwargs(self, **kwargs):
         """
         vargs = dict()
         if settings.AWX_CLEANUP_PATHS:
-            vargs['file_pattern'] = '/tmp/{}*'.format(JOB_FOLDER_PREFIX % '*')
+            vargs['file_pattern'] = '{}/{}*'.format(settings.AWX_ISOLATION_BASE_PATH, JOB_FOLDER_PREFIX % '*')
```

Comment:
We should use `os.path.join`, because we don't know if the user's value for the setting has a trailing `/`

```python
>>> os.path.join('/home/alan/', 'awx_123')
'/home/alan/awx_123'
```

So you see that solves the problem of removing the extra `/`, whether or not I add it.